packages/build-ui/src/pages/agits/_id/management/components/common_tables.rs

```diff
@@ -0,0 +1,246 @@
+use crate::pages::agits::_id::management::collectors::models::Activity;
+use crate::pages::agits::_id::management::collectors::models::Asset;
```

Comment:
It may be written in incorrect path.
models and components should be placed in the highest parent or global component.

`/pages/agits/_id/management` is parent of `pages/agits/_id/management/collectors`

So `Activity` and `Asset` should've been placed in `pages/agits/_id/management`.

And it should be exported as `self` to be used by children modules as below

```
mod model;

pub(self) mod model::*;
```